src/properties/practicebot/intake.properties

```diff
@@ -0,0 +1,2 @@
+x=4;
+y=3;
```

Comment:
The semicolons don't really matter, but we haven't been using them in other files, so let's leave them out of this as well.